logstash-core/api/lib/app/pipeline/stats_command.rb

```diff
@@ -7,14 +7,9 @@ def run
     #return whatever is comming out of the snapshot event, this obvoiusly
     #need to be tailored to the right metrics for this command.
     stats = service.get(:pipeline_stats)
-    snapshot = { "events" => {} }
-    stats.each_pair do |key, value|
-      if key.start_with?("events_")
-        field = key.gsub("events_", "")
-        snapshot["events"][field] = value
-      end
-    end
-    snapshot
+    { "events" => { "in" => stats[:base][:events_in].value, "filtered" => stats[:base][:events_filtered].value } }
```

Comment:
no specific reason, in LS I always saw strings, not symbols so this is why I stick with strings, but can use symbols :+1: